drivers/periph_common/Kconfig.timer

```diff
@@ -5,13 +5,13 @@
 # directory for more details.
 #
 
+if TEST_KCONFIG
+
 menuconfig MOD_PERIPH_TIMER
     bool "Timer peripheral driver"
     depends on HAS_PERIPH_TIMER
     select MOD_PERIPH_COMMON
 
-if MOD_PERIPH_TIMER
```

Comment:
This dependency was OK, we should keep it. All the following modules depend on `MOD_PERIPH_TIMER`.

Reply:
addressed